src/main/kotlin/uk/gov/justice/digital/hmpps/assessrisksandneeds/services/SupplementaryRiskService.kt

```diff
@@ -71,12 +71,11 @@ class SupplementaryRiskService(
 
   fun SupplementaryRiskDto.toSupplementaryRiskEntity(): SupplementaryRiskEntity {
     return SupplementaryRiskEntity(
-      supplementaryRiskUuid = this.supplementaryRiskId,
```

Comment:
We should disregard any passed in ID from a client as we should set the UUID here. This is a side effect of reusing the DTO

Reply:
Where are we actually generating a random uuid one?

Reply:
In the entity now, so it gets created when the entity is created